extension/cgroupruntimeextension/go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib/extension/cgroupruntimeextension
 
-go 1.22.0
+go 1.22.4
```

Comment:
Initially, I thought that the component's Go version needed to updated all at once, but it seems there are some components already on `go 1.22.7`. :+1: